.idea/libraries/org_bitcoinj_bitcoinj_core_0_14_3.xml

```diff
@@ -0,0 +1,20 @@
+<component name="libraryTable">
```

Comment:
An welcher Stelle und in welcher Form schlöägst Du vor, die Bibliotheken und Lizenzen aufzulisten?